solutions/04-usePaginationFragment/src/Feed.tsx

```diff
@@ -49,17 +50,20 @@ const Feed = (props: Props) => {
     if (isLoadingNext) {
       return;
     }
-    loadNext(1);
+    loadNext(10);
   }, [isLoadingNext, loadNext]);
 
   return (
     <Flex flexDirection='column'>
-      {posts.edges.map(({ node }) => (
-        <Post key={node.id} post={node} />
-      ))}
-      <Button mt='10px' onClick={loadMore}>
-        Load More
-      </Button>
+      <InfiniteScroll
+        pageStart={0}
+        loadMore={loadMore}
+        hasMore={posts.pageInfo.hasNextPage}
+        loader={<Loading />}
+        useWindow
```

Comment:
`pageStart={0}` and `useWindow` as true are default values https://github.com/danbovey/react-infinite-scroller#props
we can omit them